source/compatibility.tex

```diff
@@ -231,6 +231,19 @@
 std::format("{}", t.bit);       // ill-formed, previously returned \tcode{"0"}
 \end{codeblock}
 
+\diffref{format.formatter.spec}
+\change
+Removed the \tcode{formatter} specialization
+\begin{codeblock}
+template<size_t N> struct formatter<const charT[N], charT>.
```

Comment:
Replaced "." with ";" and added as a separate commit.

Reply:
That's not actually a valid declaration (because there's no partial specialization forward declaration, IIRC), so a semicolon is misleading. The paper correctly ends it in a period, which I suggested removing because now it's a code block.

I think there is a precedent for this.

Reply:
Oh, I guess it's fine. That form is used all over the place.
![1677591904](https://user-images.githubusercontent.com/21071787/221872191-81ae7d31-7e19-44f5-b0dd-b3e81c6969a5.png)